torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -20,6 +20,19 @@ bool normalizeOpAliases(graph_node_list_iterator& iter) {
   return false;
 }
 
+// Normalize rsub such that `rsub(x,y) = sub(x,y)`
+bool normalizeRSub(graph_node_list_iterator& iter) {
+  ArrayRef<Value*> args = iter->inputs();
+  if (iter->kind() == aten::rsub) {
```

Comment:
we all overloads for any particular symbol, so it's possible unlikely that there are other `aten::rsub` operators with different typed inputs for which this substitution isnt valid. just to be safe and future proof we should check the full schema here. you can use the `node->matches("rsub.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor")` api. 

See https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L4880, for this operator what we're doing isnt valid